src/Ng/Catalog2Elfie.cs

```diff
@@ -13,52 +13,40 @@ namespace Ng
 {
     public class Catalog2Elfie
     {
-        async Task Loop(Catalog2ElfieOptions options, CancellationToken cancellationToken)
+        /// <summary>
+        /// Creates the elfie indexes.
+        /// </summary>
+        async Task CreateIndex(Catalog2ElfieOptions options, CancellationToken cancellationToken)
         {
-            Func<HttpMessageHandler> handlerFunc = CommandHelpers.GetHttpMessageHandlerFactory(options.Verbose, null, null);
-
             // The NuGet service catalog root.
-            Uri nugetCatalogUri = new Uri(options.Source);
+            Uri nugetServicesUri = new Uri(options.Source);
```

Comment:
The variable is used on line 31.

Just to let you know, there's still a fair amount of dead code in the project. As far as I know, there's no unused variables. However, there are unused methods and a few unused types. Once we stop churning, I'll send out a PR to delete these. I didn't do it this time because the PR was already too long (sorry about that,) and I didn't want to make it more confusing.